guide/portuguese/algorithms/sorting-algorithms/bucket-sort/index.md

```diff
@@ -15,32 +15,31 @@ Vamos dar uma olhada mais de perto.
 Considere que é necessário criar uma matriz de listas, ou seja, de buckets. Os elementos agora precisam ser inseridos nesses buckets com base em suas propriedades. Cada um desses buckets pode ser classificado individualmente usando o Insertion Sort.
 
 ### Pseudocódigo para o tipo de balde:
+
 ```
-void bucketSort(float[] a,int n) 
- 
- { 
- 
-    for(each floating integer 'x' in n) 
- 
-    { 
- 
+void bucketSort(float[] a,int n)
```

Comment:
@CleoAguiar, the style guide indicates that spacing should be done with `2 spaces`.